bleak/backends/bluezdbus/scanner.py

```diff
@@ -165,7 +169,7 @@ async def start(self):
         )
         assert_reply(reply)
 
-    async def stop(self):
+    async def suspend(self):
```

Comment:
so `stop` just became `suspend`? :-D

Reply:
No, `suspend` is called inside of stop, like how it was done [here](https://github.com/workaroundgmbh/bleak/blob/859f7b966bbabdff29908e07e534f09e8b49ee40/bleak/backends/bluezdbus/discovery.py#L256)